src/services/db/GitHubService.ts

```diff
@@ -507,15 +508,16 @@ export default class GitHubService {
   async getTree(
     sessionData: UserWithSiteSessionData,
     githubSessionData: GithubSessionData,
-    { isRecursive }: { isRecursive: any }
-  ) {
+    { isRecursive }: any,
+    isStaging = true
```

Comment:
Can we standardise to use `branchName`?

Reply:
this had quite a bit of fanout :sadge: deprio for this release lets revisit this since non fnctional?